From 5bd79cad9e76397831afcee1925fc2a31d8fca8c Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 15 Feb 2020 21:55:17 +0100
Subject: [PATCH 1/1] arm: mvebu: drivers/ddr: remove redundant assignment

The value of local variable ecc is immediately overwritten. So we can
remove the first assignment.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 drivers/ddr/marvell/axp/ddr3_init.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/ddr/marvell/axp/ddr3_init.c b/drivers/ddr/marvell/axp/ddr3_init.c
index 13df9126e7..30ad5d4151 100644
--- a/drivers/ddr/marvell/axp/ddr3_init.c
+++ b/drivers/ddr/marvell/axp/ddr3_init.c
@@ -434,10 +434,6 @@ static u32 ddr3_init_main(void)
 	scrub_size = 0;
 #endif
 
-#if defined(ECC_SUPPORT) && defined(AUTO_DETECTION_SUPPORT)
-	ecc = DRAM_ECC;
-#endif
-
 #if defined(ECC_SUPPORT) && defined(AUTO_DETECTION_SUPPORT)
 	ecc = 0;
 	if (ddr3_check_config(BUS_WIDTH_ECC_TWSI_ADDR, CONFIG_ECC))
-- 
2.25.0

